### ISSUE TITLE
Can Fosscord be made LibreJS compilant?

### ISSUE THREAD

**Issue by LordRishav** [fosscord/fosscord#78]
GNU LibreJS is a Mozilla Firefox addon which filters out non-free and non-trivial JavaScript from any website. As Fosscord is already Free Software, licensed under the AGPL, can it be possible to make it LibreJS compliant? The GNU project has also published some guides on how to free your Javascript [here](https://www.gnu.org/software/librejs/free-your-javascript.html) and [here](https://www.gnu.org/software/librejs/manual/html_node/Setting-Your-JavaScript-Free.html). Thank you.

**Comment by xNaCly** [comment]
There is no need for that cause we will write all our code ourself and publish it to GitHub under the gpl license.
The code you're currently mentioning is only found the the testclient which is not the final product.

**Comment by Flam3rboy** [comment]
He is taking about the licenses of the modules we are using.

**Comment by LordRishav** [comment]
Yes, but just specifying a known license won't do any harm I guess.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
